src/signatures.ts

```diff
@@ -50,6 +52,9 @@ export async function signChallengeMessage(
   const channelId = getChannelId(signedStates[0].state.channel);
   const challengeHash = hashChallengeMessage({largestTurnNum, channelId});
 
-  const signature = await wallet.signMessage(arrayify(challengeHash));
+  // We use `web3.eth.accounts` to sign as all ethers.js signing methods are async
+  const flatSignature = new Web3EthAccounts('').sign(challengeHash, privateKey);
```

Comment:
Can this file define its own method including this logic? Then you can remove the repeated comment.